python/tchannel/tornado/tchannel.py

```diff
@@ -51,6 +56,12 @@ class State(IntEnum):
 class TChannel(object):
     """Manages inbound and outbound connections to various hosts."""
 
+    _SCHEMES = {
+        'raw': scheme.RawArgScheme,
+        'json': scheme.JsonArgScheme,
```

Comment:
Can we create a class containing fields to represent 'raw' , 'json' .. types?
I think ask use to type string input is dangerous. They may unsure it is capitalized or spell error.